pkg/crc/validation/validation.go

```diff
@@ -95,8 +95,28 @@ func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
 		if bundlePath == constants.GetDefaultBundlePath(preset) {
 			return nil
 		}
-		return ValidateBundlePath(bundlePath, preset)
+
+		if err := ValidateBundlePath(bundlePath, preset); err != nil {
+			return err
+		}
+
+		bundleInfo, err := bundle.GetBundleInfoFromName(bundle.GetBundleNameWithoutExtension(bundleName))
```

Comment:
maybe you can directly pass `bundleName` here, since `GetBundleNameWithoutExtension()` is also called in `GetBundleInfoFromName`?

Reply:
yes, good point. I added the call to GetBundleInfoFromName as a preventive measure.